src/docs/product/accounts/sso/okta-sso.mdx

```diff
@@ -11,49 +11,98 @@ description: "Set up Okta Single Sign-On (SSO) on Sentry."
 1. From the admin console's sidebar, click **Applications > Applications**, then click on "Add Application".
 
 1. Search for `Sentry` and click "Add", and you will be brought to the setup wizard.
```

Comment:
seems like some whitespace changes were made by pre commit, assuming this is fine.